devnotes/scat/README.md

```diff
@@ -0,0 +1,52 @@
+## cat for obmc-console
+
+Server of [obmc-console] used by Talos 2 communicates with its client using
+Unix socket with abstract address (see `man 7 unix`).  BMC doesn't have much
+tools installed and it's not clear if there are any for sockets with abstract
+addresses, so here's a tiny C program to get the output (socket name is
+hard-coded at the moment).
+
+It's purpose is to collect logs without a tty, so that termination of ssh
+connection wouldn't interrupt anything.
+
+### Prebuilt binary
+
+For convenience, binary is provided right here to avoid building toolchain.
+
+### Building from sources
+
+Target system uses armv6 which is not supported by popular prebuilt toolchains
```

Comment:
@SergiiDmytruk You can have a selection of up-to-date prebuilt toolchains for various architectures here: https://toolchains.bootlin.com/

Reply:
Oh, I see.